src/System.Drawing.Common/tests/Drawing2D/PathGradientBrushTests.cs

```diff
@@ -0,0 +1,919 @@
+
+            using (PathGradientBrush brush = new PathGradientBrush(_defaultFloatPoints))
+            {
+                brush.InterpolationColors = new ColorBlend() { Colors = expectedColors, Positions = expectedPositions };
+                Assert.Equal(expectedColors, brush.InterpolationColors.Colors);
+                Assert.Equal(expectedPositions, brush.InterpolationColors.Positions);
+
+                brush.InterpolationColors = new ColorBlend() { Colors = sameColors, Positions = expectedPositions };
+                Assert.Equal(sameColors, brush.InterpolationColors.Colors);
+                Assert.Equal(expectedPositions, brush.InterpolationColors.Positions);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void InterpolationColors_CannotChange()
+        {
+            Color[] colors = new Color[2] { Color.FromArgb(255, 0, 0, 255), Color.FromArgb(255, 255, 0, 0) };
+            Color[] defaultColors = new Color[1] { Color.Empty };
+
+            using (PathGradientBrush brush = new PathGradientBrush(_defaultFloatPoints))
+            {
+                brush.InterpolationColors.Colors.ToList().AddRange(colors);
+                Assert.Equal(defaultColors, brush.InterpolationColors.Colors);
+                brush.InterpolationColors.Colors = colors;
+                Assert.Equal(defaultColors, brush.InterpolationColors.Colors);
+                brush.InterpolationColors.Colors[0] = Color.Pink;
+                Assert.NotEqual(Color.Pink, brush.InterpolationColors.Colors[0]);
+                Assert.Equal(defaultColors, brush.InterpolationColors.Colors);
+                brush.InterpolationColors.Positions = new float[0];
+                Assert.Equal(1, brush.InterpolationColors.Positions.Length);
+                brush.InterpolationColors.Positions = new float[2];
+                Assert.Equal(1, brush.InterpolationColors.Positions.Length);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void InterpolationColors_Null_ThrowsNullReferenceException()
+        {
+            using (PathGradientBrush brush = new PathGradientBrush(_defaultFloatPoints))
+            {
+                Assert.Throws<NullReferenceException>(() => brush.InterpolationColors = null);
+            }
+        }
+
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
+        public void InterpolationColors_NullColors_ThrowsNullReferenceException()
+        {
+            using (PathGradientBrush brush = new PathGradientBrush(_defaultFloatPoints))
+            {
+                Assert.Throws<NullReferenceException>(() => brush.InterpolationColors = new ColorBlend() { Colors = null, Positions = null });
```

Comment:
nit: Some of these lines are a bit long. Can you put some of these onto multiple lines, if they are over ~120 characters?